hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/DBCheckpointMetrics.java

```diff
@@ -40,8 +40,10 @@ public class DBCheckpointMetrics {
   // Metrics to track checkpoint statistics from last run.
   private @Metric MutableGaugeLong lastCheckpointCreationTimeTaken;
   private @Metric MutableGaugeLong lastCheckpointStreamingTimeTaken;
+  private @Metric MutableGaugeLong lastCheckpointStreamingNumSSTExcluded;
```

Comment:
@Xushaohong , what's the main usage of this metrics?

Reply:
@ChenSammi 
1. This could be one dimension of data, providing precise info about the snapshot process. Since the log file could be flushed, the metrics might be collected and persisted.
2. For test purposes, it could be helpful instead of capturing log output.